modules/custom/reservation/js/reservation.js

```diff
@@ -25,5 +25,20 @@ function SelectingMovieOnClick() {
       
 }
 
+  function validateForm() {
+    $("#formName").submit(function(){;
+      let word = $('#customer_name').val();
```

Comment:
Zasto ovde let, a ispod var?

Reply:
Iskreno ne znam, ja mislim da sam pokusavao da mi proradi ono od juce da podigne automatski prvo slovo, ali si onda rekao da ne treba to da radim sa validacijama. Pa mi je vrv ostalo. Promenicu sada u var